webview/src/plots/components/ribbon/styles.module.scss

```diff
@@ -27,6 +26,11 @@
     display: inline;
     font-size: 0.8125rem;
   }
+
+  .fetching {
```

Comment:
[F] Gives us the space to display an icon against the experiment (e.g `<Error/>` if we have an issue with it)